CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ New functionality that are backwards-compatible:
 * [OLMIS-7444](https://openlmis.atlassian.net/browse/OLMIS-7444): Add edit modal to lot management page
 * [OLMIS-7452](https://openlmis.atlassian.net/browse/OLMIS-7452): Add ORDER_CREATE right
 * [OLMIS-7501](https://openlmis.atlassian.net/browse/OLMIS-7501): Add CCE_INVENTORY_TRANSFER right
-
+* [OLMIS-7604](https://openlmis.atlassian.net/browse/OLMIS-7604): Added valid destination page to the administration panel
```

Comment:
This is added to already released version - it shouldn't. Please create a new header for WIP version which is here: https://github.com/OpenLMIS/openlmis-referencedata-ui/blob/master/project.properties#L1
(header should be similar to this one: https://github.com/OpenLMIS/openlmis-stockmanagement-ui/blob/master/CHANGELOG.md)